ci/cloudbuild/builds/lib/quickstart.sh

```diff
@@ -45,14 +45,17 @@ function quickstart::build_cmake_and_make() {
     io::log_h2 "Building quickstart: ${lib}"
     io::log "[ CMake ]"
     src_dir="${PROJECT_ROOT}/google/cloud/${lib}/quickstart"
-    bin_dir="${PROJECT_ROOT}/cmake-out/quickstart-${lib}"
-    cmake -H"${src_dir}" -B"${bin_dir}" "-DCMAKE_PREFIX_PATH=${prefix}"
-    cmake --build "${bin_dir}"
+    cmake_bin_dir="${PROJECT_ROOT}/cmake-out/quickstart-cmake-${lib}"
+    cmake -H"${src_dir}" -B"${cmake_bin_dir}" "-DCMAKE_PREFIX_PATH=${prefix}"
+    cmake --build "${cmake_bin_dir}"
 
     echo
     io::log "[ Make ]"
+    makefile_bin_dir="${PROJECT_ROOT}/cmake-out/quickstart-makefile-${lib}"
+    mkdir -p "${makefile_bin_dir}"
     PKG_CONFIG_PATH="${prefix}/lib64/pkgconfig:${prefix}/lib/pkgconfig:${PKG_CONFIG_PATH:-}" \
       make -C "${src_dir}"
```

Comment:
What do you think of changing the Makefile to say something like:

```
BIN=.

....

$(BIN)/quickstart: ...
```

and then pass `BIN=${cmake_bin_dir}` as an argument to Make?  I would think it is better to never store the files in the source rather than move them, maybe?

Reply:
That seems better. My make-fu is rusty, so I'm glad you gave me some pointers here.

Reply:
Done. PTAL.